openid-4-verifiable-presentations-1_0.md

```diff
@@ -261,6 +268,12 @@ This specification defines the following new parameters:
 
 A public key to be used by the Wallet as an input to the key agreement to encrypt Authorization Response (see (#jarm)). It MAY be passed by the Verifier using the `jwks` or the `jwks_uri` claim within the `client_metadata` or `client_metadata_uri` request parameter.
 
+`request_uri_method`: 
+: OPTIONAL. A string determining the HTTP method to be used with the `request_uri` included in the same request. Two case-insensitive values are defined for `request_uri_method` in this specification: `get` and `post`. If `request_uri_method` is `get`, the Wallet MUST send the request to retrieve the Request Object using the HTTP GET method, i.e., as defined in [@RFC9101]. If `request_uri_method` is `post`, a supporting Wallet MUST send the same request using the HTTP POST method as detailed in (#request_uri_method_post). 
+`request_uri_method` MUST only be present if the request contains a `request_uri` parameter. If the `request_uri_method` parameter is not present, the Wallet MUST process the `request_uri` as defined in [@RFC9101]. Wallets not supporting the new method `post` will send a GET request to the request URI (default behavior as defined in [@RFC9101]).  
+
+If the Verifier uses the `request_uri_method` set to `post`, it SHOULD add the `client_metadata` parameter to the authorization request to pass its capabilities. This enables the Wallet to assess the Verifier's capabilities, allowing it to transmit only the relevant capabilities through the `wallet_metadata` request parameter in the Request URI POST request. If the Verifier uses the parameter `client_id_scheme` in the Request Object, it MUST also add the same `client_id_scheme` value in the Authorization Request.   
```

Comment:
- Not sure if it has to be "SHOULD add the `client_metadata`". This is only required if the Verifier assumes the Wallet has no other method to retrieve the `client_metadata`. If the `client_id_scheme` supports client metadata resolution, then `client_metadata` parameter is not necessary. I would suggest the following: 

> SHOULD add the `client_metadata` if the combination of `client_id` and `client_id_scheme` does not allow the Wallet to retrieve the client metadata.

Does this make sense?

Reply:
why not just MAY?

Reply:
MAY is redundant since this is already the case, right?

Reply:
I would adjust the language. I will make a proposal tomorrow. It should be highlighted that client_metadata in the authz request is not required and can be obtained from different sources and still use this feature.

Reply:
Different sources, e.g., by retrieving it via entity statements using a combination of client_id and client_id_scheme.

Reply:
I modified the text so the Verifier should add the parameter only if there is no other way. I did not want to tie it t the client id scheme Please have a look whether the text works for you.